itou/users/admin.py

```diff
@@ -506,6 +506,12 @@ def get_fieldsets(self, request, obj=None):
         else:
             fieldsets[2] = ("Permissions", {"fields": ["is_active"]})
 
+        if obj and obj.identity_provider != IdentityProvider.DJANGO:
+            # Add allow_next_sso_sub_update just after identity_provider
+            assert fieldsets[-1][0] == "Informations"
+            assert fieldsets[-1][1]["fields"][11] == "identity_provider"
+            fieldsets[-1][1]["fields"].insert(12, "allow_next_sso_sub_update")
```

Comment:
nit: les assert sur les groupes vont bien comme ça, mais pour simplifier la gestion dans le temps, que penses-tu de simplement récupérer la position d'`identity_provider`pour insérer le champ ?

```suggestion
            identity_provider_index = fieldsets[-1][1]["fields"].index("identity_provider")
            fieldsets[-1][1]["fields"].insert(identity_provider_index + 1, "allow_next_sso_sub_update")
```